src/server/game/Handlers/CharacterHandler.cpp

```diff
@@ -2113,6 +2113,52 @@ void WorldSession::HandleCharFactionOrRaceChangeCallback(std::shared_ptr<Charact
 
     if (oldRace != factionChangeInfo->Race)
     {
+        // Delete Racial Skill
+        uint32 racialSkillId;
+        switch (oldRace)
+        {
+            case RACE_HUMAN:
+                racialSkillId = 754;
+                break;
+            case RACE_ORC:
+                racialSkillId = 125;
+                break;
+            case RACE_DWARF:
+                racialSkillId = 101;
+                break;
+            case RACE_NIGHTELF:
+                racialSkillId = 126;
+                break;
+            case RACE_UNDEAD_PLAYER:
+                racialSkillId = 220;
+                break;
+            case RACE_TAUREN:
+                racialSkillId = 124;
+                break;
+            case RACE_GNOME:
+                racialSkillId = 753;
+                break;
+            case RACE_TROLL:
+                racialSkillId = 733;
+                break;
+            case RACE_BLOODELF:
+                racialSkillId = 756;
+                break;
+            case RACE_DRAENEI:
+                racialSkillId = 760;
+                break;
```

Comment:
We don't like magic numbers. Use the predefined values in Shareddefines.h see this for example https://github.com/azerothcore/azerothcore-wotlk/blob/5bfeabde813d26cf2317e39cffe598a9d1d738ab/src/server/shared/SharedDefines.h#L2981-L2983

Reply:
https://www.azerothcore.org/wiki/cpp-code-standards#random-numbers-vs-constants

Reply:
Thanks for highlighting this. Corrected in 6edb9ad.